public/class-h5p-plugin.php

```diff
@@ -66,6 +66,13 @@ class H5P_Plugin {
    */
   protected static $settings = null;
 
+  /**
+   * Default settings for HTTP Feature Policy.
+   *
+   * @var string
+   */
+  protected static $h5p_http_feature_policy = 'accelerometer *; autoplay *; camera *; clipboard-write *; fullscreen *; geolocation *; gyroscope *; magnetometer *; microphone *;';
+
```

Comment:
It would be great if you can implement this with associative arrays that apply with a WordPress filter. Then generate the string based on the after-filtered associative array. It would give developers more flexibility to enable/disable permissions based on their site/platform.

Reply:
@kelvin-xu You're welcome to amend my pull request by sending me one of your own.

Reply:
Will do, thanks

Reply:
PR has been created for review. Thanks.